src/lib/utils/validatesInputs.tsx

```diff
@@ -58,6 +59,14 @@ export const validateFeeInput = (value: string, callback: any): void => {
     callback(response);
 };
 
+export const validateOperatorPublicKey = async (publicKey: string): Promise<boolean> => {
+        const response = await Operator.getInstance().getOperatorByPublicKey(publicKey);
+        if (response.data) {
+            return true;
+        }
+       return false;
```

Comment:
Sorry there is one more comment. You can simplify this return to be one instead of 2